spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/converter/UniProtEntryConverter.java

```diff
@@ -15,6 +15,7 @@
 import org.uniprot.core.gene.Gene;
 import org.uniprot.core.scorer.uniprotkb.UniProtEntryScored;
 import org.uniprot.core.uniprotkb.*;
+import org.uniprot.core.uniprotkb.comment.CommentType;
```

Comment:
it is shame we have to mentain two implementations.
i am wondering should we try to extract common code from store and spark?

Reply:
I think we can Chat with Jie and check if we can remove clean/remove the duplicated code from spring index.